curated_transformers/layers/embeddings.py

```diff
@@ -56,10 +56,10 @@ def __init__(
 
     def forward(self, input: Tensor) -> Tensor:
         """
-        Returns the positional embedding for the input.
+        Apply sinusoidal embeddings to the input.
```

Comment:
This change is incorrect. As it is now, it only returns the positional embedding or the input.

Reply:
Indeed! reverted.